packages/core/src/app/ovm/integer-quantifiers.ts

```diff
@@ -0,0 +1,62 @@
+import { QuantifierResult, Quantifier } from '../../types'
+
+// Helper function which returns an array of numbers, starting at start, ending at end, incrementing by 1.
+// Eg. [0, 1, 2,...end]
+const range = (start: number, end: number) =>
```

Comment:
It's pretty obvious, but might suggest adding return type `(...) : number[] => ...`

Reply:
Whoops!